Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -3,6 +3,7 @@ import * as os from 'os';
 import * as path from 'path';
 import * as taskLib from 'azure-pipelines-task-lib/task';
 import * as toolLib from 'azure-pipelines-tool-lib/tool';
+import {getFileEnding} from '../javatoolinstaller';
```

Comment:
Do we need this import? This creates cyclic dependency... Maybe it's better to move it to some separate module?